pkg/controllers/dynakube/oneagent/oneagent_reconciler.go

```diff
@@ -107,6 +115,15 @@ func (r *Reconciler) Reconcile(ctx context.Context) error {
 		return r.cleanUp(ctx)
 	}
 
+	log.Info("creating Pull Secret")
+
+	err = dtpullsecret.NewReconciler(r.client, r.apiReader, r.dynakube, r.tokens).Reconcile(ctx)
+	if err != nil {
+		log.Error(err, "could not reconcile pull secret")
+
+		return err
+	}
+
```

Comment:
what if you don't want to deploy OneAgents but only an ActiveGate ? 🙃

Reply:
That's indeed a problem, still have to add the status condition to the pull secret reconciler and write unit tests for it - And then call it a second time. 

Will do that after my vacation, as I don't want to code in another bug in a hurry now.